src/test/modelmanager_test.cpp

```diff
@@ -350,6 +350,39 @@ TEST(ModelManager, ConfigReloadingWithWrongInputName) {
     ASSERT_EQ(status, ovms::StatusCode::CONFIG_SHAPE_IS_NOT_IN_NETWORK);
 }
 
+TEST(ModelManager, ConfigReloadingWithTwoModelsWithTheSameName) {
+    const char* configWithTwoSameNames = R"({
+   "model_config_list": [
+    {
+      "config": {
+        "name": "same_name",
+        "base_path": "/tmp/models/dummy1"
+      }
+    },
+    {
+      "config": {
+        "name": "same_name",
+        "base_path": "/tmp/models/dummy2"
```

Comment:
This base path is invalid, but it does not matter because duplicate name check is done before actual model loading, right?